docs/core/testing/unit-testing-published-output.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Test Published Output with dotnet vstest
+description: Learn how to run tests on published output with the dotnet vstest command.
+keywords: MSTest, .NET, .NET Core, xUnit, test, published, output, test after publishing, run tests on dll
```

Comment:
According to @mairaw we no longer need to add the keywords.

Reply:
Deleted. Is there an alternative way to tag articles to improve search engine optimization?